### ISSUE TITLE
custom values aren't returned

### ISSUE THREAD

**Issue by obambrough** [augustocdias/vscode-shell-command#113]
**Extension version**: 1.12.2
**VS Code version**: 1.93.1 (Universal) 

When an input is configured to allow custom values, the user entered custom value isn't returned.

```
{
  "version": "2.0.0",
  "tasks": [
    {
      "label": "SelectRepo",
      "command": "echo ${input: SelectRepository}",
      "type": "shell"
    }
  ],
  "inputs": [
    {
      "id": "SelectRepository",
      "type": "command",
      "command": "shellCommand.execute",
      "args": {
        "command": "docker image ls --format '{{.Repository}}' | sort | uniq",
        "allowCustomValues": true
      }
    }
  ]
}
```

### Expected behavior

When the 'SelectRepo' task is run and I enter a custom value (e.g. apache) it should print “echo apache”

### Actual behavior

When the 'SelectRepo' task is run and I enter a custom value (e.g. apache) it doesn’t print the custom value, it prints “echo”

Note: If I do use a value from the drop-down, not a custom value, it works correctly printing “echo nginx”

**Comment by MarcelRobitaille** [comment]
Thanks for the report. I can reproduce it. I think you have a mistake here though: `${input: SelectRepository}`. There should be no space.

**Comment by MarcelRobitaille** [comment]
@obambrough should be fixed in 1.12.3. Please try it.

**Comment by obambrough** [comment]
Awesome, 1.12.3 fixed it. You're right, I somehow added a space in my example task definition that wasn't in my actual tasks.json - I've updated the issue text to fix that.
